submissions/krailoartur/HTML_CSS_practice/index.html

```diff
@@ -0,0 +1,195 @@
+                                    <div class="card-img-box">
+                                        <img class="card-img" src="./popup-icons/videos.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Videos
+                                    </div>
+                                </a>
+                                <a class="card" href="#">
+                                    <div class="card-img-box">
+                                        <img class="card-img" src="./popup-icons/search.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Search
+                                    </div>
+                                </a>
+                                <a class="card" href="#">
+                                    <div class="card-img-box">
+                                        <img class="card-img" src="./popup-icons/photos.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Photo
+                                    </div>
+                                </a>
+                                <a class="card" href="#">
+                                    <div class="card-img-box">
+                                        <img class="card-img" src="./popup-icons/apps.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Apps
+                                    </div>
+                                </a>
+                                <a class="card" href="#">
+                                    <div class="img-box">
+                                        <img class="card-img" src="./popup-icons/documents.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Document
+                                    </div>
+                                </a>
+                                <a class="card" href="#">
+                                    <div class="card-img-box">
+                                        <img class="card-img" src="./popup-icons/search.png">
+                                    </div>
+                                    <div class="card-title">
+                                        Search
+                                    </div>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </li>
+                <li> <img src="./popup-icons/bell.png" class="icon"> </li>
+                <li> <img src="./popup-icons/favicon.png" class="avatar-mini"> </li>
+           </ul>
+        </nav>
+    </body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)